### PR TITLE
use cli xml instead of xml tags

### PR DESCRIPTION
**Pull request by thorko**
the xml tags had issues on large configs.
When we pushed large configs (80k) the asr9001 router just applied a part of the config (not all of it). The cisco support directed us to this page http://www.cisco.com/c/en/us/td/docs/ios_xr_sw/iosxr_r4-0/xml/programming/guide/xl40api/xl40cli.html
So we tried to use the <CLI><Configuration> syntax to apply our config and it worked.

### CODE REVIEW

**Comment by mirceaulinic** [comment]
Hi @thorko - for some reason I missed your PR, apologies for the late reply! Let me look into it and analyze the implications. Do you use this code in production since then? What's the impact, are there any negative downsides of using this method?
Thanks for your work!

**Comment by thorko** [comment]
Hi micreaulinic,
yes we use this in production. We got this hint from a cisco technician and I can't see any downside of using this method. It could be a little bit slower, but we haven't measured it.
When we used the XML tag <Commit> the router commited only part of the config (around 13K). So we had to use the other approach, when our configuration became bigger than 13K.
Cheers
thorko

**Comment by mirceaulinic** [comment]
Thanks @thorko

**Comment by mirceaulinic** [comment]
I will include this in the next release, 0.50.

File: pyIOSXR/iosxr.py [hunk 1]
```diff
@@ -600,17 +600,17 @@ def commit_replace_config(self, label=None, comment=None, confirmed=None):
...
-            rpc_command += ' Comment="%s"' % comment
+            rpc_command += ' comment %s' % comment
         if confirmed:
             if 30 <= int(confirmed) <= 300:
-                rpc_command += ' Confirmed="%d"' % int(confirmed)
+                rpc_command += ' confirmed %d' % int(confirmed)
             else:
                 raise InvalidInputError('confirmed needs to be between 30 and 300 seconds', self)
-        rpc_command += '/>'
+        rpc_command += '</Configuarion></CLI>'
```
Comment (mirceaulinic) [inline pyIOSXR/iosxr.py:613]: @thorko can you please fix this? It should be `Configuration` instead of `Configuarion`...

**Pull request merged**